windows-agent/internal/proservices/proservices.go

```diff
@@ -94,10 +100,12 @@ func New(ctx context.Context, publicDir, privateDir string, args ...Option) (s M
 	conf.SetUbuntuProNotifier(func(ctx context.Context, token string) {
 		ubuntupro.Distribute(ctx, s.db, token)
 		landscape.NotifyUbuntuProUpdate(ctx, token)
+		cloudInit.Notify(ctx)
```

Comment:
Probably due to the callback (and so the whole closure is not covered).

Do you think we can cover those conf.SetUbuntuProNotifier and SetLandscapeNotifier relatively easily?

Reply:
Yeah, none were being covered because the callbacks were never executed (for both Pro token and Landscape config notifiers).
Exercising those callbacks are not that complicated since we have a pretty fancy registry mock.
It's not that pretty, though, because the mock `defer`s notifying the registry watcher, and I'd rather investigate the impacts of changing that at a later moment.
Let me know what you think about the last commit (deferring loading files and requiring them to match the goldens).